.github/workflows/build.yml

```
@@ -183,9 +185,12 @@ jobs:
   sign:
     needs: [ build, validate-packages ]
     runs-on: windows-latest
-    if: |
-      github.event.repository.fork == false &&
-      startsWith(github.ref, 'refs/tags/')
+    #if: |
+    #  github.event.repository.fork == false &&
+    #  startsWith(github.ref, 'refs/tags/')
```

Restore once signing tested:

```suggestion
    if: |
      github.event.repository.fork == false &&
      startsWith(github.ref, 'refs/tags/')
```